src/browser/navigation-panel.js

```diff
@@ -128,10 +130,11 @@ define((require, exports, module) => {
         className: 'urlinput',
         style: theme.urlInput,
         placeholder: 'Search or enter address',
-        value: inputValue,
+        value: webViewerCursor.get('userInput'),
```

Comment:
No. You lose the ability to come back to no selection at all. We need to be able to do that:
```
type "a"
See "amazon.com, apple.com"
key down
See "amazon.com" in the input
key up
Back to "a" in the urlbar
```